src/zenml/zen_stores/secrets_stores/sql_secrets_store.py

```diff
@@ -253,7 +253,7 @@ def _check_sql_secret_scope(
 
         return False, ""
 
-    @track(AnalyticsEvent.CREATED_SECRET)
+    @track(AnalyticsEvent.CREATED_SECRET, v2=True)
```

Comment:
There is also "update_secret" and "delete_secret". Why are you targeting only "create_secret" here for v2 tracking ?

Furthermore, every secrets store implementation has its version of these methods, where events are also tracked.

Reply:
I must have not merged `develop` into my branch in time, so I missed the other implementations. If they are overriding this function, I will have to adjust those as well.